archaius2-core/src/main/java/com/netflix/archaius/ConfigProxyFactory.java

```diff
@@ -377,13 +376,13 @@ private static <T> Function<Object[], T> createDefaultMethodSupplier(Method meth
         };
     }
 
-    protected <T> MethodInvoker<T> createInterfaceProperty(String propName, final T proxy) {
-        LOG.debug("Creating interface property `{}` for type `{}`", propName, proxy.getClass());
+    protected <T> MethodInvoker<T> createInterfaceProperty(String propName, final T proxy, Class<?> targetClass) {
```

Comment:
So, basically, this method exists just so that 1) we have a neat place to log, and 2) the internal API remains symmetrical with scalar and parametrized properties?
I'm not objecting, just making sure I'm getting it :-)

Reply:
oh... actually now that I see it's a protected method, I likely shouldn't change the signature at all. I can move the logging outside of the method I suppose.